Code/Framework/AzToolsFramework/AzToolsFramework/UI/PropertyEditor/EntityPropertyEditor.cpp

```diff
@@ -2701,6 +2722,7 @@ namespace AzToolsFramework
         }
 
         AfterPropertyModified(fieldNode);
+        SetPropertyEditingComplete(fieldNode);
```

Comment:
When does ContextMenuActionPullFieldData() get called? Why are we only adding the EditingComplete() call here instead of in AfterPropertyModified()?

Reply:
Same as above.